package.json

```
@@ -122,6 +122,7 @@
     "react-native-ble-plx": "3.1.2",
     "**/babel-runtime/regenerator-runtime": "^0.13.8",
     "@metamask/transaction-controller/@metamask/controller-utils": "^5.0.0",
+    "yargs-parser": "~21.1.1",
```

```suggestion
    "yargs-parser": "^21.1.1",
```

We're already breaking majors by forcing v21 for packages expecting v11... Why not be a bit less strict in the pinning? ^^